src/Nethermind/Nethermind.AccountAbstraction/AccountAbstractionPlugin.cs

```diff
@@ -211,7 +289,15 @@ public Task InitNetworkProtocol()
             if (_accountAbstractionConfig.Enabled)
             {
                 if (_nethermindApi is null) throw new ArgumentNullException(nameof(_nethermindApi));
-                if (UserOperationPool is null) throw new ArgumentNullException(nameof(UserOperationPool));
+
+                //TODO: try filling the _userOperationPools instead and try using that
+                IDictionary<Address, UserOperationPool> _Pools = new Dictionary<Address, UserOperationPool>(); 
```

Comment:
question: why are the pools initialized here and not in the `Init` method. It feels wrong that this initialization code is repeated here and in `InitRPCModules`, and `Init` seems like the place for this kind of initializations unless there's a particular reason for not doing it there.